.ci/scripts/prepare-benchmarks-env.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+# This particular configuration is required to be installed in the baremetal
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+command -v nvm
+
+nvm install node
+nvm use node
```

Comment:
AFAIK you don't need to call `use` after `install` as the latest installed version is automatically used

Reply:
got it